src/Dialogs/Settings/dlgConfigInfoboxes.cpp

```diff
@@ -247,9 +247,9 @@ InfoBoxesConfigWidget::Prepare(ContainerWindow &parent,
   button_style.TabStop();
 
   const auto &button_look = GetLook().button;
-  copy_button.Create(parent, button_look, _("Copy"), layout.copy_button,
+  copy_button.Create(parent, button_look, _("Copy \n Set"), layout.copy_button,
```

Comment:
Please don't try to format the text of the button. 
If it overflows the button's label rendering should take care. 
Its also bad for translations.